src/waveform/renderers/allshader/digitsrenderer.cpp

```diff
@@ -5,18 +5,21 @@
 #include <QGraphicsBlurEffect>
 #include <QGraphicsPixmapItem>
 #include <QGraphicsScene>
-#include <QOpenGLTexture>
 #include <QPainter>
 #include <QPainterPath>
 #include <cmath>
 
 #include "util/assert.h"
 #include "util/roundtopixel.h"
-#include "waveform/renderers/allshader/matrixforwidgetgeometry.h"
-#include "waveform/renderers/allshader/vertexdata.h"
+#include "rendergraph/context.h"
+#include "rendergraph/geometry.h"
+#include "rendergraph/material/texturematerial.h"
+#include "rendergraph/vertexupdaters/texturedvertexupdater.h"
 
 // Render digits using a texture (generated) with digits with blurred dark outline
 
+using namespace rendergraph;
+
 namespace {
 
 // The texture will contain 12 characters: 10 digits, colon and dot
```

Comment:
```suggestion
 /// The texture will contain 12 characters: 10 digits, colon and dot
```
And elsewhere, so that it gets picked up by doxygen and IDEs.

Reply:
Is there a clear policy on this? IMO this is just a comment to help understand the code, not really API doc.

Reply:
I mean we don't really do API docs because we don't give any stability guarantees.

We don't really have a policy but some time ago we said that we try to use triple slash where it makes sense (i.e. if we add comments to classes, functions or constants). That makes the comment accessible to IDEs/LSP and people can see it when hovering over a method/constant, so that they have the explanation at hand instead of having to scroll to the definition.

Just to clarify: we said that we don't want the full-blown `@param` syntax (also hard to maintain), just that we add an additional slash so that we get IDE support for free.

Reply:
Ok. I prefer to do a follow-up PR to review all allshader code and change to /// where appropriate.